index.html

```diff
@@ -20,89 +20,115 @@ <h1 class="animated fadeInUp">YourHaq</h1>
-      <div class="col-sm-6 col-md-3">
-        <div class="service-item">
-          <h4><a href=https://www.cbp.gov/>Customs and Border Protection</a></h4>
-          <p>Missions include detecting and preventing the illegal entry of persons and goods into the country.</p>
-        </div>
-      </div>
-      <div class="col-sm-6 col-md-3">
-        <div class="service-item">
-          <h4><a href=https://www.uscis.gov/>United States Citizenship and Immigration Services</a></h4>
-          <p>The USCIS centralized FOIA office receives, tracks, and processes all USCIS FOIA requests to ensure transparency within our agency.</p>
-        </div>
-      </div>
-      <div class="col-sm-6 col-md-3">
-        <div class="service-item">
-          <h4><a href=https://www.reddit.com/r/immigration/>Immigration subreddit</a></h4>
-          <p>A place to discuss US and Worldwide immigration news, politics, visas, green cards, raids, deportations, etc.</p>
-        </div>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-sm-6 col-md-3">
-        <div class="service-item">
-          <h4><a href=http://www.law.cuny.edu/academics/clinics/immigration/clear.html>CUNY School of Law</a></h4>
-          <p>CLEAR (Creating Law Enforcement Accountability & Responsibility) aims to address the unmet legal needs of Muslim, Arab, South Asian, and others affected by counter-terrorism practices.</p>
-        </div>
-      </div>
-      <div class="col-sm-6 col-md-3">
-        <div class="service-item">
-          <h4><a href=http://www.adc.org/wp-content/uploads/2017/01/Muslim-Ban-Breakdown-Final.pdf>ADC</a></h4>
-          <p>Breakdown, written by American-Arab Anti-Discrimination Committee (ADC), of the executive order to restrict immigration from predominantly muslim nations.</p>
-        </div>
-      </div>
-      <div class="col-sm-6 col-md-3">
-        <div class="service-item">
-          <h4><a href=https://docs.google.com/spreadsheets/u/1/d/158Q5m2FtAMwq_FFoTYprhNmvpVfgVHdVQGv3CBGVMUo/htmlview#>Chicago Legal Resources</a></h4>
-          <p>Contact information of lawyers and organization representatives in Chicago, as compiled by the Muslim Women's Alliance</p>
-        </div>
-      </div>
-      <div class="col-sm-6 col-md-3">
-        <div class="service-item">
-          <h4><a href=https://onejustice.org/immigration/>OneJustice Immigration Pro-Bono Response</a></h4>
-          <p>Allows lawyers to sign-up to bring free legal services to immigrants throughout California.</p>
-        </div>
-      </div>
+    <div id="js-list">
+      <div class="row">
+        <div class="section-title">
+          <h2>Resources</h2>
+          <p>Seeking legal help? Check out the following resources.</p>
+          <label class="search-container">
+            <input class="search" placeholder="Search">
+            <i class="ion-search search-icon"></i>
+          </label>
+        </div>
+      </div>
+      <ul class="list resource-list">
```

Comment:
The `list` class is what `List` uses to locate elements to add to the search list.